### DISCUSSION TITLE
Calling API from within a commercial product

### DISCUSSION THREAD

**Discussion by pmdirector** [Boavizta/boaviztapi#225] [Q&A] [answered]
Is it permissible to call the API from within a commercial product? If yes, what are the disclosure requirements?

**Comment by da-ekchajzer** [comment, marked as answer]
You can access the licence here : https://github.com/Boavizta/boaviztapi/blob/main/LICENSE

Short answer : 

* Yes, but any modification (data or code) should be republished in the same conditions 
* You must explicitly mention the use of the API in your product.
* We strongly encourage API users to participate in its development.
